packages/asset-uploader/src/utils/mime.ts

```diff
@@ -17,24 +17,38 @@ const mimeCategories = [
 
 type MimeCategory = (typeof mimeCategories)[number];
 
-const extensionToMime = new Map([
-  [".avif" as string, "image/avif"],
-  [".bmp", "image/bmp"],
+// https://developers.cloudflare.com/images/image-resizing/format-limitations/
+const imageExtensionsToMime = new Map([
+  [".avif", "image/avif"],
   [".gif", "image/gif"],
-  [".ico", "image/vnd.microsoft.icon"],
+  [".ico", "image/x-icon"],
```

Comment:
correct mime type for ico
https://stackoverflow.com/questions/13827325/correct-mime-type-for-favicon-ico